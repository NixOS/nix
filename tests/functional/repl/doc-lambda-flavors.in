:l doc-comments.nix
:doc nonStrict
:doc strict
:doc strictPre
:doc strictPost
