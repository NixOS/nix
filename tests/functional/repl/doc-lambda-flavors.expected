Nix <nix version>
Type :? for help.

nix-repl> :l doc-comments.nix
Added <number omitted> variables.

nix-repl> :doc nonStrict
Function `nonStrict`\
  … defined at /path/to/tests/functional/repl/doc-comments.nix:37:70

My syntax is not strict, but I'm strict anyway.

nix-repl> :doc strict
Function `strict`\
  … defined at /path/to/tests/functional/repl/doc-comments.nix:38:63

I don't have to be strict, but I am anyway.

nix-repl> :doc strictPre
Function `strictPre`\
  … defined at /path/to/tests/functional/repl/doc-comments.nix:40:48

Here's one way to do this

nix-repl> :doc strictPost
Function `strictPost`\
  … defined at /path/to/tests/functional/repl/doc-comments.nix:41:53

Here's another way to do this
